testsuite/features/support/constants.rb

```diff
@@ -355,8 +355,66 @@
     sle-module-containers15-sp4-updates-x86_64
     sle-module-basesystem15-sp4-updates-x86_64
     sle-module-server-applications15-sp4-updates-x86_64
+    sle-module-public-cloud15-sp4-updates-x86_64
+    sle-module-public-cloud15-sp4-pool-x86_64
   ]
 }.freeze
 
 MIGRATE_SSH_MINION_FROM = '15-SP3'.freeze
 MIGRATE_SSH_MINION_TO = '15-SP4'.freeze
+
+OS_REPOS_BY_OS_VERSION = {
+  '11-SP4' =>
+  %w[
+    SLE-Module-Basesystem11-SP4-Pool
+    SLE-Module-Basesystem11-SP4-Updates
+    SLE-Module-Containers11-SP4-Pool
+    SLE-Module-Containers11-SP4-Updates
+  ],
```

Comment:
We don't really need SLES11 anymore, as we don't test it anymore.
But ok, let's not mix PR goals, fine to keep this :)